wallet/.gitignore

```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+/public/lockdown.umd.js
```

Comment:
this one got in. please do remove it and massage gitignore so it doesn't get in again